Exception thrown during stop

Using elasticsearch:5.3.3 and embedded-elasticsearch:2.4.1 the following exception is thrown when stop() is called.

```
	private EmbeddedElastic elastic;

	@Before
	public void before() throws IOException, InterruptedException {
		elastic = EmbeddedElastic.builder().withElasticVersion("5.5.3").withStartTimeout(1, MINUTES).build().start();
	}

	@After
	public void after() {
		elastic.stop();
	}
```

```
Exception in thread "EmbeddedElsHandler" pl.allegro.tech.embeddedelasticsearch.EmbeddedElasticsearchStartupException: java.io.IOException: Stream closed
	at pl.allegro.tech.embeddedelasticsearch.ElasticServer.lambda$startElasticProcess$7(ElasticServer.java:89)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.IOException: Stream closed
	at java.io.BufferedInputStream.getBufIfOpen(BufferedInputStream.java:170)
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:283)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:345)
	at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:284)
	at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:326)
	at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:178)
	at java.io.InputStreamReader.read(InputStreamReader.java:184)
	at java.io.BufferedReader.fill(BufferedReader.java:161)
09:21:21.258 [ElsInstanceCleaner] INFO  p.a.t.e.ElasticServer - Stopping elasticsearch server...
	at java.io.BufferedReader.readLine(BufferedReader.java:324)
	at java.io.BufferedReader.readLine(BufferedReader.java:389)
	at pl.allegro.tech.embeddedelasticsearch.ElasticServer.lambda$startElasticProcess$7(ElasticServer.java:84)
	... 1 more
```

--- a/core/src/main/java/pl/allegro/tech/embeddedelasticsearch/ElasticServer.java
+++ b/core/src/main/java/pl/allegro/tech/embeddedelasticsearch/ElasticServer.java
@@ -81,7 +81,7 @@ class ElasticServer {
                 }
                 BufferedReader outputStream = new BufferedReader(new InputStreamReader(elastic.getInputStream(), UTF_8));
                 String line;
-                while ((line = outputStream.readLine()) != null) {
+                while ((line = readLine(outputStream)) != null) {
                     logger.info(line);
                     parseElasticLogLine(line);
                 }
@@ -90,6 +90,15 @@ class ElasticServer {
             }
         }, "EmbeddedElsHandler");
         ownerThread.start();
+    }
+
+    private String readLine(BufferedReader outputStream) {
+        try {
+            return outputStream.readLine();
+        } catch (IOException e) {
+            // TODO: catching all types of IOException is not clean, any better solution?
+            return null;
+        }
     }
 
     private void installExitHook() {

